kusto/test/etoe/etoe_test.go

```diff
@@ -2258,6 +2259,7 @@ func createStringyLogsData() string {
 
 func executeCommands(client *kusto.Client, database string, commandsToRun ...kusto.Statement) error {
 	for _, cmd := range commandsToRun {
+		time.Sleep(100 * time.Millisecond)
```

Comment:
?

Reply:
can remove - again - same problem i have with testing against my cluster - can you remind  me what to change in the cluster policies to achive it